recipe_scrapers/naturallyella.py

```diff
@@ -0,0 +1,16 @@
+from ._abstract import AbstractScraper
+from ._grouping_utils import group_ingredients
+
+
+class NaturallyElla(AbstractScraper):
+    @classmethod
+    def host(cls):
+        return "naturallyella.com"
+
+    def ingredient_groups(self):
+        return group_ingredients(
+            self.ingredients(),
+            self.soup,
+            ".wprm-recipe-ingredient-group h5",
+            ".wprm-recipe-ingredient-group li",
+        )
```

Comment:
```suggestion
        )

    def total_time(self):
        total_time = self.schema.total_time()
        if total_time:
            return total_time
        return FieldNotProvidedByWebsiteException

```

Since `total_time` is a mandatory field and is _sometimes_ provided by this site we can raise an exception in cases where it is not.

Reply:
I made updates per this request, but they are slightly different than your suggestion. `self.schema.total_time()` throws a `SchemaOrgException` when `total_time` doesn't exist.

Reply:
Good catch, also upon reviewing the exception again it should only be used in instances where the field is not provided by **any** recipes on the site. I'm going to remove this sugestion and instead you can just add a `total_time: null,` to the applicable recipe.